cpu/rp2350/cpu.c

```diff
@@ -0,0 +1,58 @@
+/*
+ * SPDX-FileCopyrightText: 2025 Tom Hert <git@annsann.eu>
+ * SPDX-FileCopyrightText: 2025 HAW Hamburg
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+/**
+ * @ingroup         cpu_rp2350
+ * @{
+ *
+ * @file
+ * @brief           CPU initialization implementation for the RP2350
+ *
+ * @author          Tom Hert <git@annsann.eu>
+ */
+
+#include "cpu.h"
+
+#include "RP2350.h"
+#include "board.h"
+#include "kernel_init.h"
+#include "macros/units.h"
+#include "periph/gpio.h"
+#include "periph/init.h"
+#include "periph_cpu.h"
+#include "stdio_base.h"
+
+#define DEBUG_WITH_OSC
+
+void gpio_reset(void) {
+  reset_component(RESET_PADS_BANK0, RESET_PADS_BANK0);
+  reset_component(RESET_IO_BANK0, RESET_IO_BANK0);
+
+  /* Reenable the LED0 pin
```

Comment:
```suggestion
  /* Re-enable the LED0 pin
```

To make Codespell happy.